Dockerfile

```diff
@@ -1,5 +1,11 @@
-from python:2.7-onbuild
+FROM alpine:3.7
```

Comment:
`FROM python:2.7-alpine` is simpler no? that or `FROM python:2-7-alpine3.7 is you really want 3.7